pkgs/tools/games/ukmm/default.nix

```diff
@@ -6,27 +6,28 @@
 , wrapGAppsHook3
 , libglvnd
 , libxkbcommon
+, openssl
 , nix-update-script
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "ukmm";
-  version = "0.12.1";
+  version = "0.13.0";
 
   src = fetchFromGitHub {
     owner = "NiceneNerd";
     repo = pname;
     rev = "refs/tags/v${version}";
-    hash = "sha256-YnF0gn2JihZKkDBwI6Odne2CW8k2trQJiPbxMrtI8Gg=";
+    hash = "sha256-IhVpsDdFpeE27GY+2iZyAmXjS6F8S4ME09SneQCSuZg=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
```

Comment:
About this, I remember something about abolishing cargo lock files:

https://github.com/NixOS/nixpkgs/issues/327063

Reply:
Right now it's not possible to use `fetchCargoTarball` for newsflash:

```
ERROR: The Cargo.lock contains git dependencies

This is currently not supported in the fixed-output derivation fetcher.
Use cargoLock.lockFile / importCargoLock instead.
```